### PR TITLE
Release/update wsl readme

### PR DESCRIPTION
**Pull request by carmelp16**

### CODE REVIEW

File: docs/DOCKER.md [hunk 1]
```diff
@@ -62,8 +62,12 @@ So if the developer key is "12345" the new url need to be is: "https://maps.goog
 **For Windows:**
 You have two options: 
 Use [WSL](https://docs.microsoft.com/en-us/windows/wsl/about) OR install ubuntu VM (See [UBUNTU_VM_ON_WINDOWS](UBUNTU_VM_ON_WINDOWS.md)).
+
 For WSL - see also [ubuntu for wsl](https://ubuntu.com/wsl)
 
+***As already mentioned in [“getting the code” section](https://github.com/hasadna/anyway#getting-the-code), if you decide to use WSL for your
```
Comment (idosavion) [inline docs/DOCKER.md:68]: FYI: I think you could've used relative link

**Review by idosavion — APPROVED** [review]
thanks :)

**Comment by idosavion** [comment]
and squash

**Review by ziv17 — APPROVED** [review]
Good work.
I think it is worthwhile to mention also the following 

1. Docker is installed on Windows (and with WSL2, actually runs on the virtual environment that WSL runs on). It should run in order for docker in wsl to work.
2. Another issue I had was [Docker-credential-desktop.exe executable file not found](https://forums.docker.com/t/docker-credential-desktop-exe-executable-file-not-found-in-path-using-wsl2/100225) error. I resolved it by adding the docker Windows executable directory to the WSL path (`/mnt/c/Program Files/Docker/Docker/resources/bin`)

**Pull request merged**